packages/core/src/logs/envelope.ts

```diff
@@ -1,41 +1,47 @@
 import type { DsnComponents } from '../types-hoist/dsn';
 import type { SdkMetadata } from '../types-hoist/sdkmetadata';
-import type { SerializedOtelLog } from '../types-hoist/log';
-import type { OtelLogEnvelope, OtelLogItem } from '../types-hoist/envelope';
+import type { SerializedLog } from '../types-hoist/log';
+import type { LogContainerItem, LogEnvelope } from '../types-hoist/envelope';
 import { dsnToString } from '../utils-hoist/dsn';
 import { createEnvelope } from '../utils-hoist/envelope';
 
 /**
- * Creates OTEL log envelope item for a serialized OTEL log.
+ * Creates a log container envelope item for a list of logs.
  *
- * @param log - The serialized OTEL log to include in the envelope.
- * @returns The created OTEL log envelope item.
+ * @param items - The logs to include in the envelope.
+ * @returns The created log container envelope item.
  */
-export function createOtelLogEnvelopeItem(log: SerializedOtelLog): OtelLogItem {
+export function createLogContainerEnvelopeItem(items: Array<SerializedLog>): LogContainerItem {
   return [
     {
-      type: 'otel_log',
+      type: 'log',
+      item_count: items.length,
+      content_type: 'application/vnd.sentry.items.log+json',
+    },
+    {
+      items,
     },
-    log,
   ];
 }
 
 /**
  * Creates an envelope for a list of logs.
  *
+ * Logs from multiple traces can be included in the same envelope.
+ *
  * @param logs - The logs to include in the envelope.
  * @param metadata - The metadata to include in the envelope.
  * @param tunnel - The tunnel to include in the envelope.
  * @param dsn - The DSN to include in the envelope.
  * @returns The created envelope.
  */
-export function createOtelLogEnvelope(
-  logs: Array<SerializedOtelLog>,
+export function createLogEnvelope(
```

Comment:
l: just to confirm - log envelop headers don't need to contain the `trace` header?

Reply:
yup - more info about this here: https://www.notion.so/sentry/Log-Envelopes-Dynamic-Sampling-1d68b10e4b5d8061a035dc8bc5057196